mopidy_spotify/browse.py

```diff
@@ -104,6 +110,28 @@ def _browse_toplist_regions(variant):
             uri=f"spotify:top:{variant}:everywhere", name="Global"
         ),
     ]
+    if variant in ("tracks", "artists"):
+        dir_contents.insert(
+            0,
+            models.Ref.directory(
+                uri=f"spotify:top:{variant}:user", name="Personal"
+            ),
+        )
+    return dir_contents
+
+
+def _browse_toplist_user(webclient, variant):
+    if not webclient.logged_in:
+        return []
+
+    if variant in ("tracks", "artists"):
+        items = webclient.get_one(f"me/top/{variant}").get("items", [])
+        if variant == "tracks":
+            return list(translator.web_to_track_refs(items, False))
```

Comment:
This translator looks perfect for a `*` to force kwargs, so that you would have to put a name in front of the now very naked and not very explanatory `False`.